pact/provider/assertj/src/main/java/org/arquillian/algeron/pact/provider/assertj/ProviderUrlAssert.java

```diff
@@ -0,0 +1,19 @@
+package org.arquillian.algeron.pact.provider.assertj;
+
+import java.net.URL;
+
+public class ProviderUrlAssert {
```

Comment:
I'm not sure how those two are related. Why would we have `verifiesContract` and `satisfiesContract` next to each other?